process_manager/templates/process_manager/index.html

```diff
@@ -53,14 +53,12 @@
                type="search"
                name="search"
                placeholder="Search..."
-               hx-get="{% url 'process_manager:process_table' %}"
                hx-trigger="input changed delay:500ms, every 1s"
-               hx-target="div.table-container"
-               hx-swap="innerHTML">
+               hx-get="{% url 'process_manager:process_table' %}"
+               hx-target="#table-container">
         <div id="table-container"
-             hx-get="{% url 'process_manager:process_table' %}"
-             hx-swap="innerHTML"
-             hx-trigger="load"></div>
+             hx-trigger="load"
+             hx-get="{% url 'process_manager:process_table' %}"></div>
```

Comment:
What are these changes for? Diff is a bit confusing as some lines are being moved around. Could you tidy it a bit to minimise the diff?

Reply:
Diff was a bit messy, but just removed `innerHTML` as it's the default anyway. Fixed the diff.